docs/user_guide.md

```diff
@@ -32,35 +60,35 @@ Read the instructions of the agent to see what it does.
 Feel free to modify the instructions to see how it affects the agent's behavior.
 See if you can make it a soccer expert for instance!
 
-We'll describe the structure of agent networks' .hocon files in next section.
+We'll describe the structure of agent networks' `.hocon` files in next section.
 
 ## Hocon files
 
 ### Import and Substitution
 
 HOCON files support importing content from other HOCON files using the unquoted keyword `include`, followed by whitespace and the path to the imported file as a quoted string:
-```json
+```hocon
```

Comment:
GitHub was not rendering the syntax properly because it's not valid `.json` syntax. But it's valid `.hocon` syntax and GitHub happily renders it we specify it that way.